runtime/common/src/migration_helper.rs

```diff
@@ -0,0 +1,31 @@
+// substrate
+pub use frame_support::migration;
+
+/// Pallet migration helper.
+pub struct PalletCleaner {
+	pub name: &'static [u8],
+	pub values: &'static [&'static [u8]],
+	pub maps: &'static [&'static [u8]],
+}
+impl PalletCleaner {
+	/// Remove all storage from a pallet.
+	pub fn remove_all(&self) -> u32 {
+		self.remove_storage_values() + self.remove_storage_maps()
+	}
+
+	/// Remove multiple storage value from a pallet at once.
+	pub fn remove_storage_values(&self) -> u32 {
+		self.values.iter().for_each(|i| {
+			let _ = migration::clear_storage_prefix(self.name, i, &[], None, None);
+		});
+
+		self.values.len() as u32
```

Comment:
Are you sure that the write times for the clearing are the same as the `self.values.len()`? For example, If there is a pallet storage map with 100 elements, the write times is only 1 instead of 100?

Reply:
Hmm, this is `remove_storage_values`, it makes sense that way.